src/scripting/ScriptContext.cpp

```diff
@@ -16,10 +17,58 @@ ScriptContext::ScriptContext(sol::state_view aStateView, const std::filesystem::
             m_onUpdate = aCallback;
         else if(acName == "onDraw")
             m_onDraw = aCallback;
-        else if(acName == "onConsoleOpen")
-            m_onConsoleOpen = aCallback;
-        else if(acName == "onConsoleClose")
-            m_onConsoleClose = aCallback;
+        else if(acName == "onToolbarOpen")
+            m_onToolbarOpen = aCallback;
+        else if(acName == "onToolbarClose")
+            m_onToolbarClose = aCallback;
+        else
+            Logger::ErrorToModsFmt("Tried to register unknown handler '{}'!", acName);
+    };
+
+    m_env["registerVKBind"] = [this](const std::string& acID, const std::string& acDescription, sol::table aVKBindCode, sol::function aCallback)
```

Comment:
This wasn't what we talked about, also the name is too specific, users are not supposed to know how we implement that

Reply:
well maybe resolved dunno :P